src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -0,0 +1,423 @@
+/*
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge;
+
+import org.jitsi.impl.neomedia.*;
+import org.jitsi.impl.neomedia.rtp.*;
+import org.jitsi.service.neomedia.*;
+import org.jitsi.util.*;
+
+import java.lang.ref.*;
+import java.util.*;
+import java.util.concurrent.*;
+
+/**
+ * Implements a hack for
+ * https://bugs.chromium.org/p/chromium/issues/detail?id=403710. The hack
+ * injects black video key frames to unstuck the playback of audio for composite
+ * media streams.
+ *
+ * @author George Politis
+ */
+public class LipSyncHack
+{
+    /**
+     * A byte array holding a black key frame.
```

Comment:
This is the full UDP payload of an RTP packet that contains a VP8 key frame. Added some clarifying JavaDoc.